src/util/journalling_symbol_table.h

```diff
@@ -100,6 +100,11 @@ class journalling_symbol_tablet : public symbol_table_baset
     return result;
   }
 
+  std::size_t next_unused_suffix(const std::string &prefix) const override
```

Comment:
A comment about the commit message: I don't understand "Due to the inclusion of another method we just need to make sure any calls are routed to the wrapped table and not the parent." Which method is this "another" method that's causing trouble?